src/coreComponents/mesh/CellElementSubRegion.cpp

```diff
@@ -42,24 +47,71 @@ CellElementSubRegion::CellElementSubRegion( string const & name, Group * const p
 
 CellElementSubRegion::~CellElementSubRegion()
 {
-  // TODO Auto-generated destructor stub
+  // Left blank
 }
 
-void CellElementSubRegion::copyFromCellBlock( CellBlock & source )
+void CellElementSubRegion::setElementType( ElementType elementType )
```

Comment:
Since this function is very similar to `CellBlock::setElementType`, is it really needed? Can you just directly copy the `m_elementType`, `m_numNodesPerElement`, `m_numEdgesPerElement`, `m_numFacesPerElement` from the `CellBlock` to the `CellElementSubRegion`?

Reply:
I simplified.